### PR TITLE
Added carousel to the home page

### PR DESCRIPTION
**Pull request by ericm**
I added this using angular-bootstrap and custom css overrides. It's highly configurable and modular so adding and changing slides will be easy.
# Screenshot:

### CODE REVIEW

File: src/app/home/home.component.css [hunk 1]
```diff
@@ -29,3 +29,36 @@
...
+  text-align: left;
+  padding: 1em;
+  font-size: 2em;
+  top: calc(15vh - 69px);
+  height: 110px;
+  width: 60%;
+  border-bottom: 5px solid #e91e63a8;
+  color: #151515;
+  left: 10%;
+  width: 800px;
```
Comment (CaelumF) [inline src/app/home/home.component.css]: overwriting width above it?

**Comment by CaelumF** [comment]
UL image source https://hea.ie/higher-education-institutions/university-of-limerick/governance/

**Comment by ericm** [comment]
# Updated screenshot

**Comment by CaelumF** [comment]
If you're not entirely sure where the other image comes from then remove it, but apart from that it looks good so will approve after that

**Comment by ericm** [comment]
@CaelumF I fixed those issues

**Comment by ericm** [comment]
Image source: https://en.wikipedia.org/wiki/College#/media/File:Corpus_Christi_College_New_Court,_Cambridge,_UK_-_Diliff.jpg

**Pull request merged**